includes/Classifai/Admin/SavePostHandler.php

```diff
@@ -58,12 +58,13 @@ public function did_save_post( $post_id ) {
 			return;
 		}
 
-		$supported   = \Classifai\get_supported_post_types();
-		$post_type   = get_post_type( $post_id );
-		$post_status = get_post_status( $post_id );
+		$supported     = \Classifai\get_supported_post_types();
+		$post_type     = get_post_type( $post_id );
+		$post_status   = get_post_status( $post_id );
+		$post_statuses = $this->get_post_statuses( $post_type, $post_id );
```

Comment:
To keep things consistent, I'd recommend we handle this the same way we get the supported post types. So instead of a method within the `SavePostHandler` class, add it as a function in our `Helpers.php` file.

Reply:
@dkotter This has been updated to use a similar method to get the post statuses that the existing get post types function, new helper functions have been added to accommodate this.